components/docs/Sidebar/Dropdown.jsx

```diff
@@ -36,12 +36,14 @@ export default function Dropdown({
         {routes.routes.map((r, idx) => {
           if (!r.path) {
             return (
-              <Dropdown
-                routes={r}
-                parentOpen={open}
-                key={`${r.title}-${idx}`}
-                setSidebarCollapsed={setSidebarCollapsed}
-              />
+              <li key={`${r.title}-${idx}`}>
+                  <Dropdown
+                    routes={r}
+                    parentOpen={open}
+                    key={`${r.title}-${idx}`}
```

Comment:
This key is now double, not sure if that is a good/ bad thing.

Reply:
I wasn't sure either :joy:  But it worked fine for me in testing so I assumed it wouldn't be a huge problem!